.flake8

```diff
@@ -0,0 +1,5 @@
+# flake8 configuration.
+[flake8]
+exclude = .tox,.git,__pycache__
+ignore = D211,D400,E251,E731,W503
```

Comment:
`E251 - unexpected spaces around keyword / parameter equals`  Can be fixed easily (??)

Reply:
Yes but it is massive change, so I assumed it was better to leave it.

However I can add that, preferable in a later PR.